pkg/controller/quay_io_ci_images_distributor/mirror.go

```diff
@@ -100,7 +101,9 @@ func (c *MirrorConsumerController) Run(ctx context.Context) error {
 			continue
 		}
 		if len(mirrors) == 0 {
-			c.logger.Debug("Waiting for mirror tasks ...")
+			if !c.isPjrehearse {
```

Comment:
I didn't want to have the logs of  `Waiting for mirror tasks ...` in the `pj-rehearse` it is disturbing there